jest.config.js

```diff
@@ -7,9 +7,13 @@ const supportsESMAndJestLightRunner = semver.satisfies(
   "^12.22 || ^13.7 || >=14.17"
 );
 const isPublishBundle = process.env.IS_PUBLISH;
+const isDebug = process.env.TEST_DEBUG;
 
 module.exports = {
-  runner: supportsESMAndJestLightRunner ? "jest-light-runner" : "jest-runner",
+  runner:
+    supportsESMAndJestLightRunner && !isDebug
+      ? "jest-light-runner"
```

Comment:
I use `jest-light-runner` when debugging without problems :thinking: Why do we need to disable it?

Reply:
I can't hit any debuggers inside test setup or tests when it's turned on. I don't know why that is.

Reply:
What OS and node version are you using?

Reply:
Oh, looking at https://github.com/nodejs/node/issues/26609 it seems like it doesn't work with some debuggers, such as Chrome.

You may add a comment that if your debugger does not support worker_threads, you can edit the jest config to use the old runner.

Reply:
Ah yep, I'm using chrome's. I don't like telling people to change configs. If you're not comfortable with keying this on `TEST_DEBUG` maybe it could have its own environment variable like `TEST_NO_WORKER_THREADS` which contributing could mention?

Reply:
Nah, I'll just implement same-thread support in the light runner.

Reply:
Jest uses same-thread execution when it determines that only one file matches `testNamePattern`. Will you use that same heuristic?

Reply:
No, for simplicity you will always have to specify the `--runInBand` option (which is implied by `TEST_DEBUG`).

Could you try if https://github.com/nicolo-ribaudo/jest-light-runner/pull/20 works for you? You only have to replace `jest-light-runner`'s version in `package.json` with `nicolo-ribaudo/jest-light-runner#run-in-band`.

Reply:
It works I hit my breakpoint.